### DISCUSSION TITLE
請問老師是否每天都會刷題?

### DISCUSSION THREAD

**Discussion by CHANG-CHING-CHUNG** [Lidemy/forum#29] [其他問題]
頭香

**Comment by aszx87410** [comment]
AoC 每天都會寫，邊開題邊吃午餐，吃完午餐解法差不多就想出來了，寫完就可以睡午覺了
除此之外沒在寫題目

**Reply by CHANG-CHING-CHUNG** [reply]
難道是傳說中的午餐解題法?(吃一吃頭頂上突然冒出一個燈泡大喊，我想到了!!)XDD
加贈一題:
老師覺得刷 leetcode 等等題目，有助於寫程式嗎?

**Reply by aszx87410** [reply]
有啊，就是訓練解題思考的能力

**Reply by CHANG-CHING-CHUNG** [reply]
老師有刷過 leetcode 嗎? 若有，總共刷了多少題呢?，又，哪些題目讓你印象深刻

**Reply by aszx87410** [reply]
沒有XDD 以前無聊的時候寫過兩三題就沒繼續了 提不起勁來寫

**Reply by CHANG-CHING-CHUNG** [reply]
那，老師有玩過 baba is you 嗎？是一種超燒腦解謎遊戲，據我朋友所說，他以前玩過的解題遊戲都比不上這一款😂

**Reply by aszx87410** [reply]
好像有玩過一點點 覺得好難就關掉了XD

**Reply by CHANG-CHING-CHUNG** [reply]
真的很難，一般關卡都還好，extra 超難QQ
